controllers/ironic_controller.go

```diff
@@ -210,6 +211,9 @@ func (r *IronicReconciler) SetupWithManager(mgr ctrl.Manager) error {
 		Owns(&batchv1.Job{}).
 		Owns(&corev1.Secret{}).
 		Owns(&corev1.ConfigMap{}).
+		Owns(&corev1.ServiceAccount{}).
```

Comment:
I think all we want is to add these to all the controllers.

Reply:
Fixed in the latest version